<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="device_tab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Devices</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QPushButton" name="reset_button">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="apply_button">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Apply</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QTreeView" name="device_view">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>DejaVu Sans Mono</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::CurrentChanged|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideMiddle</enum>
          </property>
          <property name="indentation">
           <number>20</number>
          </property>
          <property name="rootIsDecorated">
           <bool>true</bool>
          </property>
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <attribute name="headerShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="headerStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="permanent_checkbox">
          <property name="text">
           <string>Permanently</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="message_tab">
       <attribute name="title">
        <string>Messages</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="messages_text">
          <property name="font">
           <font>
            <family>Monospace</family>
            <pointsize>8</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settings_tab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" colspan="2">
         <widget class="QToolBox" name="toolBox">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>198</width>
             <height>126</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Notifications</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="1" column="0">
             <widget class="QCheckBox" name="notify_allowed">
              <property name="text">
               <string>Allowed</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QCheckBox" name="notify_removed">
              <property name="text">
               <string>Removed</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="notify_rejected">
              <property name="text">
               <string>Rejected</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="notify_inserted">
              <property name="text">
               <string>Inserted</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="2">
             <widget class="QCheckBox" name="notify_blocked">
              <property name="text">
               <string>Blocked</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="notify_ipc">
              <property name="text">
               <string>IPC status</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QCheckBox" name="notify_present">
              <property name="text">
               <string>Present</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>487</width>
             <height>246</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Device Dialog</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="4" column="0" colspan="3">
             <widget class="QCheckBox" name="decision_permanent_checkbox">
              <property name="text">
               <string>Decision is permanent by default</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="3">
             <widget class="QCheckBox" name="show_reject_button_checkbox">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reject causes the device to be logically removed from the operating system. The device might still get power from the from the bus, however, it has to be reinserted for it to get visible for the operating system again.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Show the Reject button</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Default decision timeout (seconds)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Decision method</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Default decision</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QComboBox" name="default_decision_combobox">
              <item>
               <property name="text">
                <string>allow</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>block</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>reject</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSpinBox" name="decision_timeout">
              <property name="suffix">
               <string/>
              </property>
              <property name="maximum">
               <number>3600</number>
              </property>
              <property name="value">
               <number>23</number>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QComboBox" name="decision_method_combobox">
              <item>
               <property name="text">
                <string>Buttons</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QCheckBox" name="randomize_position_checkbox">
              <property name="text">
               <string>Randomize position of the window</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QCheckBox" name="mask_serial_checkbox">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Starting from the end, mask every 2nd character in the displayed serial number value with an asterisk symbol.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Mask the serial number value</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
